### DISCUSSION TITLE
Recovering account details

### DISCUSSION THREAD

**Discussion by Kriegbaum** [inventree/InvenTree#8320] [Q&A]
Hi folks. I recently moved apartments and it took me a whole 6 months to get my workshop setup again, which includes the Inventree server.
I'm finding that I never put my admin account in my password manager and I no longer remember my credentials. I looked around the documentation quite a bit and I'm not finding a way to get these details. I failed to configure the password recovery option.
Currently running in a docker container on my own server, so I have root access for all the inventree data and access to the secrets file. Do I have any options for recovering my admin credentials?
I know this is my own fault, these will go straight to the password manager once they are recovered.
Any help would be greatly appreciated!

**Comment by Kriegbaum** [comment]
I have successfully guessed my credentials which is great
I'm going to leave this post up though because it seems like the answer to this question could be useful to someone else someday

**Comment by SchrodingersGat** [comment]
@Kriegbaum glad you got it sorted out. I have an incoming PR - https://github.com/inventree/InvenTree/pull/8321 - to discuss how this could be managed in the documentation

**Reply by Kriegbaum** [reply]
Rad, that was fast!
I had a look at it, one thing I was struggling with was that I had even forgot my username, if `manage.py` also has a way to dump available usernames, listing that instruction might be a gift to someone as lost as I was, but the instructions in the PR are very helpful already
